### PR TITLE
#66 Extract camtrap contributors from datapackage.json

### PR DESCRIPTION
**Pull request by mike-podolskiy90**

### CODE REVIEW

File: crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java [hunk 1]
```diff
@@ -221,6 +245,41 @@ private static Optional<License> readLicense(Path unpackedDpPath) {
...
+  private static List<Contact> readContributors(Path unpackedDpPath) {
+    List<Contact> contacts = new ArrayList<>();
+
+    File dataPackage = Paths.get(unpackedDpPath.toString(), "datapackage.json").toFile();
+    JsonNode root = MAPPER.readTree(dataPackage);
+
+    for (JsonNode contributorNode : root.path("contributors")) {
+      Contact contact = new Contact();
+
+      setContactFieldFromJsonNode(contributorNode, "firstName", node -> contact.setFirstName(node.asText()));
```
Comment (marcos-lg) [inline crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java:258]: In the dataset that we have in UAT I see contributors like:

```
{
    "title" : "Peter Desmet",
    "path" : "https://orcid.org/0000-0002-8442-8025",
    "email" : "peter.desmet@inbo.be",
    "organization" : "Research Institute for Nature and Forest (INBO)",
    "role" : "contributor"
  }
  ```

and it doesn't match this mapping, are we sure about this mapping?

Comment (mike-podolskiy90) [reply crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java:258]: Need to check why they're not set. Those fields are properly set in the `datapackage.json` inside the archive

Comment (mike-podolskiy90) [reply crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java:258]: Those are newly added fields so were not present for some datasets maybe. I've published a new version and they're all set

Comment (mike-podolskiy90) [reply crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java:258]: Example https://ipt3.gbif-uat.org/resource?r=mica-full

**Pull request merged**